clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/op/job/KubectlJobExecutor.java

```diff
@@ -332,7 +333,8 @@ public KubernetesManifest get(KubernetesV2Credentials credentials, KubernetesKin
       command.add("-l=" + selectors.toString());
     }
 
-    JobStatus status = jobExecutor.runJob(new JobRequest(command));
+    List<KubernetesManifest> result = new ArrayList<>();
```

Comment:
if this List is being written to potentially on a different thread by the JobExecutor, does this List need to be a ConcurrentList to ensure there's no visibility edge cases with items added on the JobExecutor thread not showing up on this thread?

Reply:
Great point...I hadn't made it concurrent as it's only ever being written to by a single thread, but did not enforce any "happens-after" semantics between writing to the list in one thread and reading from it in the other.  Now that `runJob` returns the results rather than modifying a variable in the calling thread, this is no longer an issue.